package/boot/uboot-envtools/files/ramips

```diff
@@ -33,6 +33,11 @@ xiaomi,miwifi-nano|\
 zbtlink,zbt-wg2626)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x10000"
 	;;
+hootoo,ht-tm05)
+        idx="$(find_mtd_index u-boot-env)"
+        [ -n "$idx" ] && \
```

Comment:
Why would that be needed? Is there something variable about the partitioning?

Reply:
Do you propose to use `$idx` or `$(find_mtd_index u-boot-env)` directly in `ubootenv_add_uci_config` line below?

Reply:
I would have expected just
ubootenv_add_uci_config "/dev/mtd5" ...

So, what's the reason that the number would be different from that?

Reply:
I see the value using the partition name `u-boot-env`, as it is searchable, but `mtd5` is not.

For the example, let say you don't allow the introduce `CONFIG_MTD_VIRT_CONCAT=y` and one should merge the `firmware2` partition into `loader`. In that case the `u-boot-env` would be `mtd4`.

Reply:
Sorry, not following the desired / needed change here 🤣. Is something needed? Thanks!